package.json

```diff
@@ -6,9 +6,11 @@
   "scripts": {
     "lerna": "lerna",
     "bootstrap": "lerna bootstrap --concurrency 1",
+    "bootstrap-link": "npm run link-all && npm run interlink",
```

Comment:
This sounds like "bootstrap + link" but seems to been just linking. The description sounds like it should be bootstrap + link as well. Can you clarify?

Reply:
"npm link"  installs as well as symlinks into global area. So this `bootstrap-link` has lerna `npm link` each package (installing deps and linking globally) then runs the `interlink` script which has lerna symlinking the packages all together appropriately.